src/workerd/util/sqlite-kv.h

```diff
@@ -15,6 +15,11 @@ namespace workerd {
 // of KJ exceptions which become internal errors.
 class SqliteKvRegulator: public SqliteDatabase::Regulator {
   void onError(kj::Maybe<int> sqliteErrorCode, kj::StringPtr message) const override;
+
+  // We bill for KV operations as rows read/written.
+  virtual bool shouldAddQueryStats() const override {
```

Comment:
(I guess the `stmtCountKeys` prepared statement doesn't use this regulator yet, so would be excluded from stats, but not sure if that query represents notable overhead.)

Reply:
Ah good catch, probably should fix that.

Reply:
Fixed.

There was a comment saying:

```
    // We don't pass in the regulator here because this query doesn't take user input, so
    // it failing is always our fault.
```

I decided this concern doesn't really mater as there is simply no way the query would ever fail, except for deeper I/O errors that could affect any of these queries, so it might as well use the same regulator as everything else.